starknet_devnet/blueprints/base.py

```diff
@@ -48,7 +48,7 @@ def extract_positive(request_json, prop_name: str):
     return value
 
 
-def hex_converter(request_json, prop_name: str, convert=custom_int) -> int:
+def hex_converter(request_json, prop_name: str, convert=validate_hex_string) -> int:
```

Comment:
I wouldn't expect a function named `validate_...` to return something

Reply:
I'll rename this! I missed that during my own review. I just copied the method name from the task and I forgot to change that, sorry.

Reply:
renamed from 'validate_hex_string' to 'parse_hex_string' in 1dd85e8a5f0ed28166d69cf8ef144d7ecc4df94c